### PR TITLE
General fixes to install script

### PR DESCRIPTION
**Pull request by directhex**
Stop installing way more stuff than we need to

### CODE REVIEW

**Comment by radical** [comment]
We seem to be missing some files in the `Output` directory like `Microsoft.{CSharp,VisualBasic}.Core.targets`, `csc.exe`(and friends) etc. Also, `*UnitTests*` assemblies are present even in this, so, those should still be removed.

**Comment by directhex** [comment]
Do we _want_ csc.exe? Are we expecting that building msbuild from source bundles Roslyn from NuGet?

File: install-mono-prefix.sh [hunk 1]
```diff
@@ -6,26 +6,27 @@ fi
...
+cp -R nuget-support/tv/* ${DESTDIR}${XBUILD_DIR}/$MSBUILD_TOOLSVERSION
+cp -R nuget-support/tasks-targets/* ${DESTDIR}${XBUILD_DIR}/
 
-for f in $XBUILD_DIR/Microsoft/NuGet/*; do ln -s $f $XBUILD_DIR ; done
+for f in ${XBUILD_DIR}/Microsoft/NuGet/*; do ln -s $f ${DESTDIR}${XBUILD_DIR} ; done
 
 sed -e 's,@bindir@,'$MONO_PREFIX'/bin,' -e 's,@mono_instdir@,'$MONO_PREFIX/lib/mono',' msbuild-mono-deploy.in > msbuild-mono-deploy.tmp
 chmod +x msbuild-mono-deploy.tmp
-cp msbuild-mono-deploy.tmp $MONO_PREFIX/bin/msbuild
+cp msbuild-mono-deploy.tmp ${DESTDIR}${MONO_PREFIX}/bin/msbuild
```
Comment (radical) [inline install-mono-prefix.sh:32]: And add a `rm msbuild-mono-deploy.tmp`. I forgot to add that earlier!

**Comment by radical** [comment]
Other than the minor comment, LGTM! 👍

**Pull request merged**